src/fmu/dataio/_utils.py

```diff
@@ -203,11 +218,18 @@ def read_parameters_txt(pfile):
         }
     """
 
+    logger.debug("Reading parameters.txt from {}".format(pfile))
+
     with open(pfile, "r") as stream:
-        buffer = stream.read().replace(" ", ":").splitlines()
+        buffer_ = stream.read().splitlines()
```

Comment:
I don't understand this "buffer_" as the word "buffer" is not a reserved word in python(?)

Reply:
My IDE flagged it and I went on autopilot from there... Can revert it.

Reply:
Looks like Python 2.7 has it. Not present >3.5 as far as I can see. That's probably why it was flagged. 

https://docs.python.org/2.7/library/functions.html#buffer